packages/core/src/lib/lit-core.ts

```diff
@@ -488,15 +484,14 @@ export class LitCore {
     if (!this.config.contractContext) {
       this.config.contractContext = await LitContracts.getContractAddresses(
         this.config.litNetwork,
-        new ethers.providers.JsonRpcProvider(
-          this.config.rpcUrl || LIT_CHAINS['lit'].rpcUrls[0]
+        new ethers.providers.StaticJsonRpcProvider(
+          this.config.rpcUrl || RPC_URL_BY_NETWORK[this.config.litNetwork]
```

Comment:
Fixed a bug here; we were always using `LIT_CHAINS['lit']` here.  It just so happened that due to internal override logic inside of the contracts SDK this actually worked.

Reply:
Should we do this in the constructor instead? so that `this.config.rpcUrl` is always defined

Reply:
cc @MaximusHaximus  ^^